liquibase-core/src/main/java/liquibase/io/EmptyLineAndCommentSkippingInputStream.java

```diff
@@ -42,7 +42,7 @@ private int read(final int lastRead, final boolean lookAhead) throws IOException
         if (commentSkipEnabled && (read == this.commentLineStartsWith.toCharArray()[0])
                 && (lastRead == '\n' || lastRead < 0)) {
             while ((((read = super.read())) != '\n') && (read != '\r') && (read > 0)) {
-                ;//keep looking
+                //keep looking
```

Comment:
**Today I Learned**

The `;` in this loop indicated an empty statement; Java still implicitly "knows" there is an empty statement, even without the `;` character.